lib/Unicode/Char/Numeric.hs

```diff
@@ -46,13 +46,18 @@ isNumber c = case generalCategory c of
     OtherNumber   -> True
     _             -> False
 
-
 -- | Numeric value of a character, if relevant.
+--
+-- __Note:__ a character may have a numeric value but return 'False' with
+-- the predicate 'isNumber', because 'isNumber' only tests 'GeneralCategory'.
```

Comment:
We may want to put a note in `isNumber` as well.

Reply:
Done